source/geometry_model/initial_topography_model/prm_polygon.cc

```diff
@@ -100,6 +110,7 @@ namespace aspect
              * we need to fill the point lists and topography values. They
              * are stored in the Topography subsection in the Topography parameter.
              */
+            maximum_topography = 0;
```

Comment:
In the hypothetical case that all the prescribed topography is negative, this won't be seen by taking the maximum with zero. Not sure how that affects stuff down the line, but I guess it would be neatest to use something like -std::numeric_limits<double>::max().

Reply:
good point, I changed it.